pkgs/by-name/fl/flac123/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  flac,
+  libao,
+  libogg,
+  popt,
+}:
+
+stdenv.mkDerivation {
+  pname = "flac123";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "flac123";
+    repo = "flac123";
+    rev = "v2.1.1";
```

Comment:
I'd expect a string interpolation with `version` here instead of `v2.1.1`.

Reply:
Yep. You're right! Give me a minute to amend it.